hm_pyhelper/hardware_definitions.py

```diff
@@ -1,3 +1,31 @@
+import os
+
+
+def is_rockpi():
+    return 'rockpi' in os.getenv('BALENA_DEVICE_TYPE')
```

Comment:
Nit: For consistency between `is_rockpi` and `is_raspberry_pi`, should either use the full string "rockpi-4b-rk3399" here, or switch below to:

```
device_type_match = [
        'raspberrypi',
        'nebra-hnt',
    ]
```